### PR TITLE
feature/stripe-integration

### PR DESCRIPTION
**Pull request by alex94945**
Following this:
- [x] Step 1 [link here](https://rails.devcamp.com/ruby-gem-walkthroughs/payment/how-to-integrate-stripe-payments-in-a-rails-application-charges):
    - [x] Before the "Refactor" section
    - [x] Completed "Refactor" section

- [x] Step 2 (subscription) [link here](https://rails.devcamp.com/ruby-gem-walkthroughs/payment/how-to-implement-subscriptions-in-stripe):

### CODE REVIEW

**Comment by alexneigher** [comment]
this is lookin super good. What's the status on this guy?

**Comment by alex94945** [comment]
Step 1 complete. Need to do https://rails.devcamp.com/ruby-gem-walkthroughs/payment/how-to-implement-subscriptions-in-stripe next

File: app/controllers/charges_controller.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  end
+
+  private
+
+    def set_plan
+      @plan = 9999
+    end
+
+    def amount_to_be_charged
+      @amount = 500
```
Comment (alexneigher) [inline app/controllers/charges_controller.rb]: @alex94945 this is 500 cents aka $5. Isn't the plan $50?

File: app/controllers/charges_controller.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  before_action :set_description
+  before_action :authenticate_user!
+  before_action :set_plan
+
+  def new
+  end
+
+  def create
+    if params[:subscription].include? 'yes'
+      StripeTool.create_membership(email: params[:stripeEmail],
```
Comment (alexneigher) [inline app/controllers/charges_controller.rb]: what happens in this branch of the conditional? We don't create a charge or a customer, nor do we save the response of this call to a variable. What conditions must be present for this branch to be reached, and what is the desired outcome?

File: app/views/charges/new.html.haml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+= form_tag charges_path do
```
Comment (alexneigher) [inline app/views/charges/new.html.haml]: we should add some styling and messaging here re: 
1) already have an account in good standing
2) already have an account that has gone into bad standing
3) canceling/deleting/ending payments

File: app/controllers/charges_controller.rb (hunk 2 above)
Comment (alex94945) [inline app/controllers/charges_controller.rb]: hmm i don't think it's currently being used. the idea is that there's a hidden field on the charges/new page which gets set to yes if the charge is supposed to be for a subscription.

File: app/controllers/charges_controller.rb (hunk 1 above)
Comment (alex94945) [inline app/controllers/charges_controller.rb]: fixed

**Pull request merged**